examples/cloud_l4_l7_native_load_balancer/main.tf

```diff
@@ -0,0 +1,119 @@
+terraform {
+  required_providers {
+    aci = {
+      source = "ciscodevnet/aci"
+    }
+  }
+}
+
+provider "aci" {
+  username = ""
+  password = ""
+  url      = ""
+  insecure = true
+}
+
+resource "aci_tenant" "tf_tenant" {
+  name = "tf_tenant"
+}
+
+# Cloud Subnet setup part
+resource "aci_vrf" "vrf1" {
+  tenant_dn = aci_tenant.tf_tenant.id
+  name      = "vrf-1"
```

Comment:
Should we follow best practices in the naming and prevent the "-" as delimiter? Also this keeps it consistent with tenant naming above. 

[General Recommendations](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/aci/apic/sw/kb/b-Cisco-ACI-Naming-and-Numbering.html#id_107264)

Use Underscores as Delimiters

In constructing the distinguished name (DN) of an object, the ACI system uses hyphens to separate system-generated prefixes from suffixes that include user-defined naming strings. Because underscores are not used in any system-generated strings, they are ideal for use as delimiters in naming strings. In the following example, the system combines the prefix "tn-" with the user-defined tenant name "CloudMgmt_Tenant" to create the the DN. Because the naming string uses an underscore instead of a hyphen, it is easier to differentiate where the system object name component ends and where the user-defined component begins.

<fvTenant dn="uni/tn-CloudMgmt_Tenant" name="CloudMgmt_Tenant">